### PR TITLE
Add content grouping for Google Analytics.

### PR DESCRIPTION
**Pull request by loranallensmith**
This is a follow-up to #9 that adds a bit of additional configuration to our Google Analytics tracking tag so that we can segment traffic on GitHub Pages traffic more easily.

### CODE REVIEW

**Comment by loranallensmith** [comment]
@claudiosanches I wasn't sure the best way to request a review on this one, but I figured I'd ping you here because I think you're a primary maintainer on this repository (and because you reviewed the original PR too).  Let me know if there is anything else I should do for this PR before it's ready to merge.  :smiley:

**Review by claudiosanches — APPROVED** [review]

**Pull request merged**
